<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWidget</class>
 <widget class="QWidget" name="RegisterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="setAccount_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="setPassword_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setAccount_LineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="setPassword_LineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="accountRequire_Label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6-11位数字、大小写字母</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordRequire_Label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数字、大小写字母</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPassword_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPassword_LineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterBtn1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账号不符合要求</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码不符合要求</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
